RoboticonColony/Assets/Code/Tile.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+sealed public class Tile
+{
+    public int Cost { get; private set; }
+    public int OwnerID { get; private set; }
+
+    private Dictionary<ItemType, int> _Resources;
+    private int _ownerID;
```

Comment:
You wouldnt want a new instance of the owner, just pass in a reference to a player class